### PR TITLE
Basic health check

### PR DESCRIPTION
**Pull request by infofromca**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.HealthChecks/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace OrchardCore.HealthChecks
+{
+    public class Startup : StartupBase
+    {
+        public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
+        {
+            app.UseHealthChecks("/health/live");
+
+        }
```
Comment (Jetski5822) [inline src/OrchardCore.Modules/OrchardCore.HealthChecks/Startup.cs]: remove empty space

**Comment by Jetski5822** [comment]
Simple - nice. Would be interesting to see the health-check modules that spawn off this.

**Comment by sebastienros** [comment]
That's a good start. We'll need to iterate on integration with other modules, like adding more health checks as features, and configurability (the endpoint url for instance), but it's a good idea. Thanks.

**Comment by sebastienros** [comment]
Can you add a reference in the OrchardCore.Cms target? Also, remember to add the reference in alphabetical order

**Comment by sebastienros** [comment]
Next step is to provide implementations of health checks from features. Even from existing modules.

**Pull request merged**